src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -176,6 +178,21 @@ class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionCont
     }
   }
 
+  override def getProjectMembers(projectId: String)(implicit userToken: WithAccessToken): Future[Seq[RawlsBillingProjectMember]] = {
+    authedRequestToObject[Seq[RawlsBillingProjectMember]](Get(FireCloudConfig.Rawls.authUrl + s"/billing/$projectId/members"), true)
+  }
+
+  override def addUserToBillingProject(projectId: String, role: ProjectRole, email: String)(implicit userToken: WithAccessToken): Future[Unit] = {
+    val url = FireCloudConfig.Rawls.authUrl + s"/billing/$projectId/${role.toString}/${java.net.URLEncoder.encode(email, "UTF-8")}"
+
+    userAuthedRequest(Put(url), true) map { resp =>
+      resp.status match {
+        case OK => ()
+        case _ => throw new FireCloudExceptionWithErrorReport(FCErrorReport(resp))
+      }
+    }
+  }
+
```

Comment:
I generally don't like methods that return `Unit`. Could this at least return a `Boolean` of `resp.status.isSuccess`?